File: ObjectCDemo/DatingApp/DatingApp/View/EventDetailCells/EventInforTableViewCell.m
//
//  EventInforTableViewCell.m
//  DatingApp
//
//  Created by Steven Jimy   on 11/11/16.
//  Copyright © 2016 Steven Jimy . All rights reserved.
//

#import "EventInforTableViewCell.h"

@implementation EventInforTableViewCell

- (void)awakeFromNib {
    [super awakeFromNib];
    self.layerView.layer.cornerRadius = 5.0f;
    self.layerView.clipsToBounds = YES;
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

@end

File: SwiftDemos/DatingApp/DatingApp/View/EventDetailCells/EventInforTableViewCell.swift
//
//  EventInforTableViewCell.swift
//  DatingApp
//
//  Created by Steven Jimy   on 11/10/16.
//  Copyright © 2016 Steven Jimy . All rights reserved.
//

import UIKit

class EventInforTableViewCell: UITableViewCell {

    @IBOutlet weak var layerView: UIView!
    override func awakeFromNib() {
        super.awakeFromNib()
        self.layerView.layer.cornerRadius = 5
    }

    override func setSelected(_ selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)

        // Configure the view for the selected state
    }
    
}
